src/mrack/providers/provider.py

```diff
@@ -127,7 +127,9 @@ async def _wait_for_ssh(self, host, port=22, timeout=1200):
 
         return res, host
 
-    async def _provision_base(self, reqs):  # pylint: disable=too-many-locals
+    async def _provision_base(
+        self, reqs, res_check_timeout=60
```

Comment:
Could you please state in the docstring that `res_check_timeout` default is 60 minutes?

Reply:
Can do